styles/dialog.css

```diff
@@ -66,9 +66,6 @@ ai-dialog > ai-dialog-header > button {
   padding: 0;
   cursor: pointer;
 }
-ai-dialog > ai-dialog-header > button:focus {
```

Comment:
That change was introduced after I ran `gulp build`. I assumed the previous version of the file was stale but I did not investigate thoroughly.